file/src/test/java/org/apache/calcite/adapter/file/FileReaderTest.java

```diff
@@ -68,6 +68,7 @@ private static String resourcePath(String path) throws Exception {
   }
 
   /** Tests {@link FileReader} URL instantiation - no path. */
+  @Ignore("[CALCITE-3289] Wikipedia format change breaks file adapter test")
   @Test public void testFileReaderUrlNoPath() throws FileReaderException {
     Assume.assumeTrue(FileSuite.hazNetwork());
 
```

Comment:
I think it might because your internet has been restricted.

Reply:
run the testFileReaderUrlNoPath,get the Exception in my mac.
org.junit.AssumptionViolatedException: got: <false>, expected: is <true>
	at org.junit.Assume.assumeThat(Assume.java:95)
	at org.junit.Assume.assumeTrue(Assume.java:41)
Maybe we could ignore this test.